src/gordon_gcp/plugins/event_consumer.py

```diff
@@ -31,10 +31,13 @@
 """
 
 import asyncio
+import concurrent.futures
```

Comment:
Originally, in the second comment for this PR, I followed the approach from [here](https://github.com/cloudfind/google-pubsub-asyncio/blob/master/app.py). Testing it locally made me realize that it doesn't _actually_ work (if you run with `PYTHONASYNCIODEBUG=1`, there's a lot of issues going on. This commit mainly fixes that approach, with some other minor things that I noticed were wrong (i.e. inverted some args passed into google libs) as well as touched up some things (`pubsub_msg._ack_id` is actually a long string like `'projects/a-project/subscriptions/test-sub:1413'` and not a number, which isn't nice for logs and such).

While google's pubsub package allows for message polling, [it actually blocks](http://google-cloud-python.readthedocs.io/en/latest/pubsub/subscriber/index.html#subscription-callbacks) (aka the rest of the entire gordon service would be blocked). Therefore, I chose to use a [threadpool executor](https://github.com/spotify/gordon-gcp/pull/9/commits/5299d682957d7d575e2f3f2a4b849067f99613e2#diff-5d2065d110f9259f5221a7aec9ef5ff6R435) to unblock. 

The polling functionality requires a callback function (to process a message), but that too is also blocking (only non-async funcs accepted). So garnering heavy inspiration from this [stack overflow post](https://stackoverflow.com/a/29302684), I elected to implement a small wrapper around the `threading.Thread` class with its own event loop (separate from the Main event loop). If a task from another thread is handed to it – which it is, from the main event loop – it runs it in its event loop with explicit threadsafety handling (via `loop.call_soon_threadsafe`). If it's handed a task from its own thread, then it just runs it on its own loop directly (no threadsafety handling needed).